vault-admin-client/src/main/java/com/scality/osis/vaultadmin/impl/cache/CacheConstants.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2021 Scality, Inc.
+ * SPDX-License-Identifier: Apache License 2.0
+ */
+
+package com.scality.osis.vaultadmin.impl.cache;
+
+public final class CacheConstants {
+    //Env variables
+    public static final String ENV_LIST_ACCOUNT_DISABLED = "osis.scality.vault.cache.listAccounts.disabled";
+    public static final String ENV_LIST_ACCOUNT_MAX_CAPACITY = "osis.scality.vault.cache.listAccounts.maxCapacity";
+
+
+
+    // Defaults
+    public static final int DEFAULT_CACHE_MAX_CAPACITY = 1000;
+    public static final long DEFAULT_EXPIRY_TIME_IN_MS = 30000;
+    public static final int DEFAULT_SCHEDULED_THREAD_POOL_SIZE = 3;
```

Comment:
Any reason the pool size would be > 1?

The design of having 1 expire task per cache entry creates a potentially large number of work units for later (if not, then the cache wouldn't be really needed in the first place). Then having several threads compete for locks would create just more CPU usage (reentrant or not, they still have a cost).

How about having only 1 expire thread that expires a batch of eligible entries a few times per second? It should be lighter on contention since it would:
- happen less often
- happen a fixed number of times compared to once per cache item
- have less threads competing overall

It seems like it would be feasible without much added cost, since the tracking list would need to be traversed under lock (for removal of all expired entries in 1 pass) a few times per sec, instead of (worst case) hundreds of times (up to the max capacity).

Reply:
That sounds good to me. Threadpool with one thread each cacheObject makes sense but I would like to do performance evaluation later (in another ticket). For now I will change this to 1

Reply:
fixed in 6df3c84